content/en/synthetics/browser_test.md

```diff
@@ -32,31 +32,30 @@ Browser tests are scenarios executed by Datadog on your web applications. They r
 
 Define the configuration of your browser test.
 
-{{< img src="synthetics/browser_test/browser_check_configuration.png" alt="Browser Tests make request" responsive="true" style="width:80%;">}}
+1. **Starting URL**: The URL from which your browser test starts the scenario.
+    * Advanced Options (optional): Use custom request headers or cookies. For example, set the User Agent in the header to identify Datadog scripts.
```

Comment:
We should add that headers defined here will *override* the default browser headers (for instance it'll override the User Agent as you noted) but cookies will instead be *added* to the browser cookies.